content/livesync/quickstart.textile

```diff
@@ -1,7 +1,241 @@
 ---
 title: Quickstart
-meta_description: "."
+meta_description: "A quickstart guide to learn the basics of integrating the Ably LiveSync product into your application."
 product: livesync
 languages:
   - javascript
 ---
+
+This quickstart will provide you with a set of instructions to get an example running locally on your machine. Following this, you'll cover each component of LiveSync found within the code of the example.
+
+h2(#example). The example
+
+In this section you will cover all the steps required to get an existing example running locally on your machine.
+
+h3(#step-0). Clone the repository
+
+Clone the "models repository":https://github.com/ably-labs/models/ and navigate to the @livecomments@ directory within @examples@. The @livecomments@ directory houses the project used for this example.
```

Comment:
This needs to be updated now that the example has its own repo: https://github.com/ably-labs/live-comments, along with all the other links to source in this document